README.md

```diff
@@ -22,35 +22,49 @@ To have a better understanding about the API, please make sure to read the gener
 ## Installation
 
 For a simple installation you can use [Composer](https://getcomposer.org/).
-Since we are _not yet_ on Packagist you might want to require this library via [Composer VCS](https://getcomposer.org/doc/05-repositories.md#vcs):
-
-```json
-{
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/TheKeymaster/findologic-api"
-        }
-    ],
-    "require": {
-        "TheKeymaster/findologic-api": "dev-master"
-    }
-}
-```
+Using this command will install the latest version.
 
-Usually the `master` branch should be pretty stable, but this will change in the future as soon as we have our first real stable version.
+```bash
+composer require thekeymaster/findologic-api
+```
 
 ## Basic usage
 
 The usage is pretty simple. Here is an example:
 
 ```php
-// Examples will be added in a future release.
+// Require composer autoload
+require_once __DIR__ . '/vendor/autoload.php';
+
+use FINDOLOGIC\Api\Config;
+use FINDOLOGIC\Api\RequestBuilders\Xml\SearchRequestBuilder;
+use FINDOLOGIC\Api\Client;
+
+$config = new Config();
+// ServiceId/Shopkey, you can find it in the customer account.
+$config->setServiceId('ABCDABCDABCDABCDABCDABCDABCDABCD');
+
+// Client used for requests
+$client = new Client($config);
+
+$searchRequest = new SearchRequestBuilder();
+$searchRequest
+    ->setQuery('shirt') // Users search query
+    ->setShopurl('blubbergurken.de') // Url of the shop
+    ->setUserip('127.0.0.1') // Users IP
+    ->setReferer('https://shop.url/AGB') // Page where search was fired

```

Comment:
From the [PHP documentation](https://www.php.net/manual/en/reserved.variables.server.php) (`HTTP_REFERER`):

> The address of the page (if any) which referred the user agent to the current page. This is set by the user agent. Not all user agents will set this, and some provide the ability to modify HTTP_REFERER as a feature. In short, it cannot really be trusted.

As states, it can not really be trusted since the client is responsible for sending it, but it could be used.